src/matrixrtc/MatrixRTCSession.ts

```diff
@@ -636,8 +640,17 @@ export class MatrixRTCSession extends TypedEventEmitter<MatrixRTCSessionEvent, M
         }
     };
 
+    private isMyMembership = (m: CallMembership): boolean =>
+        m.sender === this.client.getUserId() && m.deviceId === this.client.getDeviceId();
+
+    /**
+     * Examines the latest call memberships and handles any encryption key sending or rotation that is needed.
+     *
+     * This function should be called when the room members or call memberships might have changed.
+     */
     public onMembershipUpdate = (): void => {
         const oldMemberships = this.memberships;
+        const oldFingerprints = this.lastMembershipFingerprints;
```

Comment:
Nitpick: this can be moved to immediately before `this.storeLastMembershipFingerprints()`, as it's not used outside of the `if (this.manageMediaKeys ...)` block.

Reply:
Good point. Now [done](https://github.com/matrix-org/matrix-js-sdk/pull/4343/commits/2487ad6703380153f97df7fc58e9509fa18d9e27).